fastlane_bot/modes/pairwise_multi_all.py

```diff
@@ -67,6 +67,9 @@ def find_arbitrage(self, candidates: List[Any] = None, ops: Tuple = None, best_p
                 if len(base_direction_two) > 0:
                     curve_combos += [[curve] + base_direction_two for curve in not_carbon_curves]
 
+                if len(carbon_curves) >= 2:
+                    curve_combos += [carbon_curves]
```

Comment:
This causes test 064 to fail with:
```
>       assert carbon_wrong_direction_count == 0, f"[NBTest64 TestMultiPairwiseAll Mode] Expected all Carbon curves to have the same tkn in and tkn out. Mixing is currently not supported."
E       AssertionError: [NBTest64 TestMultiPairwiseAll Mode] Expected all Carbon curves to have the same tkn in and tkn out. Mixing is currently not supported.
E       assert 6 == 0
```
We can fix that specific assertion in the test, but I'd like to make sure that this is the right thing to do:
1. Should we expect carbon_wrong_direction_count == 6 here?
2. Does this mean that Mixing is currently not supported is no longer the case?

If not, then there is likely a problem with adding these two lines of code at this specific point.

Reply:
Resolved. Can support mixing of carbons, potentially all curves